core/spec.md

```diff
@@ -5528,6 +5565,36 @@ as defined below:
   - MUST NOT be present if the Resource's `hasdocument` model attribute is
     set to `false.
 
+##### `ancestor` Attribute
+- Type: String
+- Description: The `versionid` of this Version's ancestor.
+
+  The `ancestor` attribute MAY be set to any existing Version's `versionid`,
+  or to the `versionid` of the Version itself, to indicate to the server that
```

Comment:
`to indicate that this Version has no "ancestor"`.
The "to the server" implies it's some kind of signal from the client to the server, which it is sometimes, but in general we should talk about the meaning of the attribute when it appears in the datastore.
```suggestion
  or to the `versionid` of the Version itself, to indicate to the server that
```